blocks.py

```diff
@@ -137,3 +137,29 @@ def steering_angle(tank, gyro, speed, sharpness, angle):
     while gyro.angle() <= small or gyro.angle() >= big:
         tank.drive(speed, sharpness)
     tank.stop()
+    
+def check_cargo(my_robot):
```

Comment:
It would be nice to add a simple docstring on the top of a complex function describing what it does; we can add this later.